### PR TITLE
Create 2018-02-21.md

### PR DESCRIPTION
**Pull request by hackygolucky**
Add Feb 21 subcommittee approved meeting minutes

### CODE REVIEW

**Review by williamkapke — APPROVED** [review]

File: joint-strategy-subcommittee/2018-02-21.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+#### February 21, 2018
+
+#### Present:
+
+- Mark Hinkle, Executive Director, Node.js Board 
+- Sarah Novotny, Board Member, Google
+- Todd Moore, Board Member, Node.js Chairman, IBM
+- Ashley Williams, Individual Director
+- William Kapke, Individual Director,
```
Comment (ashleygwilliams) [inline joint-strategy-subcommittee/2018-02-21.md]: suprious comma

File: joint-strategy-subcommittee/2018-02-21.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+### Expectations (20 minutes)
+- The Future of Individual Membership Program(20 minutes) 
+  
+## NOTES
+
+- Reviewed the agenda and purpose for today. 
+- Points of discussion for project expectations of the Board
+  - What support is expected? What responsibilities are expected in the relationship? 
+   - Clear definition of what the Board does 
+   - Manages money
```
Comment (ashleygwilliams) [inline joint-strategy-subcommittee/2018-02-21.md]: should these be indented one more?

File: joint-strategy-subcommittee/2018-02-21.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+ 
+- What the Board does:
+  - Running the business, legal and marketing aspects of the Node.js Foundation 
+  - Prioritizing what investments to allocate funds to, and oversee a budget (e.g. fund build and test infrastructure) 
+  - Define the structure, process and terms for a Certification & Compliance program 
+  - Oversee Marketing such as PR/AR, branding via a members only marketing committee 
+  
+How do these not align?
+
+Early on in the founding, the Board spent a lot of energy on sharing information and managing expectations.
```
Comment (ashleygwilliams) [inline joint-strategy-subcommittee/2018-02-21.md]: more context on this point might help, i dont remember what it means

**Review by ashleygwilliams — COMMENTED** [review]
some very small nits

File: joint-strategy-subcommittee/2018-02-21.md (hunk 3 above)
Comment (hackygolucky) [reply joint-strategy-subcommittee/2018-02-21.md]: ah, good point. Was in re: to the mystery of what the Board does

**Pull request merged**